components/Badge/Badge.css

```diff
@@ -3,18 +3,18 @@
   flex-direction: column;
   justify-content: flex-start;
   align-items: center;
-  margin: 1rem;
+  margin: 2rem;
```

Comment:
Badge exists elsewhere in the app, so these changes affect those usages. Let's leave the stylesheet here untouched. Instead, we can override the Badge instances in the code school card with a className.

`<Badge className={styles.codeSchoolCardIcon} ...`